src/cephadm/cephadm

```diff
@@ -1704,6 +1716,27 @@ except ImportError:
                 loop.close()
 
 
+def _log_from_call(verbosity: 'CallVerbosity', message: str, error: bool = False) -> None:
```

Comment:
Maybe this is total overkill, but it may make it easier to read. Something like:
```python

class CallVerbosity(Enum):
    ....

    def error_log_level(self) -> int:
        if self == self.DEBUG:
            return logging.DEBUG
        if self == self.QUIET:
            return QUIET_LOG_LEVEL
        return logging.INFO

    def success_log_level(self) -> int:
        if self == self.VERBOSE:
            return logging.INFO
        if self in (self.QUIET, self.QUIET_UNLESS_ERROR):
            return QUIET_LOG_LEVEL
        return logging.DEBUG

def  _log_from_call(verbosity: 'CallVerbosity', lines: Iterable[str], error: bool = False) -> None:
    if error:
        level = verbosity.error_log_level()
    else:
        level = verbosity.success_log_level()
    for line in lines:
        logger.log(level, message)
```

Reply:
I like the idea. Added some functions like this to the CallVerbosity class. Tried to make it use an explicit dict rather than if statements as well so we're very explicit. It works fine, although I couldn't figure out how to make mypy happy with it, so I have a `#  type: ignore` when indexing the dict currently. The only thing from here I wasn't super into was using the iterable instead of jsut a string for the arg to `_log_from_call`. Main reason being if I did accidentally pass a string, it would just print one char per line, so I would have to enforce passing a list arg always. What I ended up with for now is just removing the `_log_from_call` function altogether because with the logic for getting the log level being elsewhere the function barely did anything anymore. Instead it now just grabs the log level using the new functions and just logs it without additional function calls. Let me know what you think.

Reply:
Works for me.  I'll review the new version now.